man/systemd.exec.xml

```diff
@@ -1305,6 +1309,14 @@
                 <entry>Debugging, performance monitoring and tracing functionality (<citerefentry project='man-pages'><refentrytitle>ptrace</refentrytitle><manvolnum>2</manvolnum></citerefentry>, <citerefentry project='man-pages'><refentrytitle>perf_event_open</refentrytitle><manvolnum>2</manvolnum></citerefentry> and related calls)</entry>
               </row>
               <row>
+                <entry>@file-system</entry>
+                <entry>Basic file system access: opening and creating files and directories for read and write, reading file properties</entry>
+              </row>
+              <row>
+                <entry>@file-system-ops</entry>
+                <entry>File system operations: more advanced file system operations, such as renaming or removing files and directories, or creating hard and symbolic links.</entry>
```

Comment:
@ronnychevalier yes this can be interesting and it goes into the direction of capsicum of FreeBSD, however here not sure if this makes sense, but instead of emulating the capabilities mode of the kernel, one can mimic partial functionality at userspace level using the file descriptor that you mention coupled with a mount namespace setup where only the fds that maps to resources and desired capabilities are available...